examples/screenshot/src/main.rs

```diff
@@ -40,6 +40,12 @@ impl eframe::App for MyApp {
                     "continuously take screenshots",
                 );
 
+                if self.continuously_take_screenshots {
+                    if ui.button("save to 'top_left.png'").clicked() {
```

Comment:
It's a bit confusing that this button is only shown in continuous mode (and it is hard to find, because in continuous mode, the screen is… weird)

Reply:
I agree. Made it constantly visible + clicking the button will request a screenshot.